packages/instantsearch.js/src/types/index.ts

```diff
@@ -22,6 +22,3 @@ export * from './widget';
 export * from './ui-state';
 export * from './render-state';
 export * from './templates';
-
-// from specific widgets
-export type { IndexWidget } from '../widgets/index/index';
```

Comment:
Could this be potentially breaking?

Reply:
it's moved to the "widgets" file, still exported from here too through that `*`

Reply:
Ah yes, I see it now.